src/main/frontend/config.cljs

```diff
@@ -374,12 +374,21 @@
   [repo-url relative-path]
   (when (and repo-url relative-path)
     (cond
-      (and (or (util/electron?) (mobile-util/is-native-platform?)) (local-db? repo-url))
+      (and (or (util/electron?) (mobile-util/native-android?)) (local-db? repo-url))
       (let [dir (get-repo-dir repo-url)]
         (if (string/starts-with? relative-path dir)
           relative-path
           (str dir "/"
                (string/replace relative-path #"^/" ""))))
+
+      (and (mobile-util/native-ios?) (local-db? repo-url))
+      (let [dir (-> (get-repo-dir repo-url)
+                    (string/replace "file:///" "file:/")
+                    (string/replace "%20" " "))]
```

Comment:
better use encodeURIComponent

Reply:
![image](https://user-images.githubusercontent.com/72891/147448810-1d5dad20-363e-42be-a070-bed11f7371c3.jpeg)
To handle more non-url-safe chars

Reply:
not decodeURIComponent?

Reply:
decodeURI

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/decodeURI